tests/integration/bond_test.py

```diff
@@ -777,3 +778,15 @@ def test_new_bond_uses_mac_of_first_slave_by_name(eth1_eth2_with_no_profile):
         assert get_mac_address(BOND99) == eth1_mac
         _nmcli_simulate_boot(BOND99)
         assert get_mac_address(BOND99) == eth1_mac
+
+
+def test_add_invalid_slave_ip_config(eth1_up):
+    desired_state = eth1_up
+    desired_state[Interface.KEY][0][Interface.IPV4][InterfaceIP.ENABLED] = True
```

Comment:
Perfer `InterfaceIPv4.ENABLED` as we already know its IPv4. Not a must.

Reply:
Fixed this, thanks!